docs/components/arm/yahboom-dofbot.md

```diff
@@ -3,20 +3,22 @@ title: "Configure a yahboom-dofbot Arm"
 linkTitle: "yahboom-dofbot"
 weight: 50
 type: "docs"
-description: "Configure a yahboom-dofbot arm."
+draft: "true"
```

Comment:
does it need to be a draft?

Reply:
Until I publish the module - I'd rather yes - also I wasn't sure if you were documenting viam-labs modules, which are meant to be less well-vetted than the ones in the viam namespace.